symbols/overlay29.yml

```
@@ -492,6 +492,10 @@ overlay29:
         r0: Entity pointer
         r1: Target position
         return: True if the specified monster can target the target position, false otherwise.
+    - name: PopulateActiveMonsterPtrs
+      address:
+        EU: 0x22E32B8
+      description: Populates active_monster_ptrs with all valid monsters in the monster_slot_headers array.
```

Do you know that there's no params? Not mentioning anything is meant for when you aren't sure. If you are, put:

```suggestion
      description: |-
        Populates active_monster_ptrs with all valid monsters in the monster_slot_headers array.
        
        No params.
```